test/test_sedcli_basic.py

```diff
@@ -27,6 +30,23 @@ def gen_rand_key(key_len):
 	letters = string.ascii_lowercase
 	return  ''.join(random.choice(letters) for i in range(key_len)).encode('ascii')
 
+def rw_opal_drive(device_path):
+	# Read from the disk
+	pipe = subprocess.Popen(["dd", "if=" + device_path, "of=/dev/null", "bs=4k", "count=100", "iflag=direct"], stdout = subprocess.PIPE, stdin = subprocess.PIPE, stderr = subprocess.PIPE)
+	err = pipe.communicate()
+	if (pipe.returncode == 0):

```

Comment:
I would expect to see here some kind of assertion checking for dd exit code with expected exit code, that is: when drive is locked I would expect that read and write should fail, thus dd command will fail. Test code should have appropriate assert calls to check it.